src/components/Article/Article.css

```diff
@@ -1 +1,32 @@
 /* Put article styles here */
+
+.article {
```

Comment:
Remove redundant code. Check it everywhere, please 
```suggestion
.article {
```